packages/legend-query-builder/style/_query-builder.scss

```diff
@@ -979,3 +979,93 @@
     padding: 2rem;
   }
 }
+
+.query-builder__export-other-viewer {
+  &__modal {
+    padding: 0;
+    height: 30vh;
```

Comment:
Perhaps add increased sizing for height so codeeditor has more space for content?